### PR TITLE
Lower Casing webpack

### PR DESCRIPTION
**Pull request by rouzbeh84**
After reading the branding guidelines I thought it be best to adhere to them and keep webpack cased as intended across the project. Thoughts?

### CODE REVIEW

**Comment by bebraw** [comment]
Yeah, it's good. Ideally we would have a linting rule (markdown lint or proselint?) against this. Much easier to handle.

**Comment by SpaceK33z** [comment]
YES FINALLY, someone who also cares about lower-cased webpack haha. Thanks!

File: content/configuration/entry-context.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ contributors:
   - tarang9211
 ---
 
-The entry object is where Webpack looks to start building the bundle. The context is an absolute string to the directory that contains the entry files.
+The entry object is where webpacklooks to start building the bundle. The context is an absolute string to the directory that contains the entry files.
```
Comment (SpaceK33z) [inline content/configuration/entry-context.md]: Made a small typo there, it should have a space.

Comment (rouzbeh84) [inline content/configuration/entry-context.md]: it's closed but i'll make a quick adjustment and submit. good catch!

Comment (rouzbeh84) [inline content/configuration/entry-context.md]: j/k hha.

**Pull request merged**